src/index.js

```diff
@@ -281,7 +285,11 @@ export default async function fetch(url, options_) {
 
 			// For gzip
 			if (codings === 'gzip' || codings === 'x-gzip') {
-				body = pump(body, zlib.createGunzip(zlibOptions), reject);
+				body = pump(body, zlib.createGunzip(zlibOptions), err => {
+					if (err) {
```

Comment:
nit pick, can you quickly change all `err` to `error` to make the linter happy?

must say that i'm not a huge fan of xo... like standardjs more...